components/MarkdownEditor.tsx

```diff
@@ -26,6 +26,7 @@ export default function MarkdownEditor(props: {
   });
   // The "text"-state is used for displaying the changing text instantly
   const [text, setText] = useState(defaultText);
+  Icon.add({ link });
```

Comment:
This can be called outside of the function since it only needs to be done once.

Reply:
ok